tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -44,20 +54,52 @@ Target GenerateVariables
                        GenerateIntegrationTestsDebuggerArm64Matrices();
                    });
 
+            bool CriticalTracerChanges(string[] changedFiles, CodeOwnersParser codeOwners)
+            {
+                // These folders are owned by @DataDog/tracing-dotnet but changes should not affect ASM functionality
+                string[] nonCriticalDirectories = new[]
+                {
+                    "tracer/test/",
+                    "tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/",
+                    "tracer/src/Datadog.Trace.", // Does not match the main tracer project
+                    "tracer/src/Datadog.Trace/Agent/",
+                    "tracer/src/Datadog.Trace/ContinuousProfiler/",
```

Comment:
I'm not sure this one would impact ASM, though it _would_ affect profiling obvs?